stages/org.osbuild.fix-bls

```diff
@@ -18,13 +18,14 @@ This stage reads and (re)writes all .conf files in /boot/loader/entries.
 STAGE_OPTS = ""
 
 
-def main(tree, _options):
+def main(tree, options):
```

Comment:
For the commit message:

I think the way this is currently done is based on a misreading of the specification. The specification does not seem to consider `/` a valid `$BOOT`, but requires it to be a separate filesystem (as it is all about sharing $BOOT between distros). In addition to the definition of the `$BOOT` variable, see also assumptions made further down in the spec, e.g., "`$BOOT/loader/` is the directory containing all files needed for Type #1 entries".

I still think this patch is what we want/need, but I think the reason is to work around `/boot` being on the rootfs, which it should not be (and we should change that in all our images, I guess. Failing that, I think the best solution medium-term would be to ship always ship and install the BLS snippets under the assumption that `/boot` is a separate filesystem, and rather pass an argument to grub to add a prefix if necessary, so we never have to fiddle with the files at install-time (this would be impossible if one day the BLS snippets are signed, for instance).